File: ResourceManager/Settings/AppSettings.cs
namespace ResourceManager.Settings
{
    public class AppSettings
    {
        public const string SectionName = "AppSettings";
        public string AppName { get; set; }
        public string AppVersion { get; set; } 
        public string AppLanguage { get; set; }
        public EbookSettings EbookSettings { get; set; }
        public ArticleSettings ArticleSettings { get; set; }
        public ActivitySettings ActivitySettings { get; set; }
        public RepositorySettings RepositorySettings { get; set; }
    }
    public class EbookSettings
    {
        public const string SectionName = "EbookSettings";
        public string FolderPath { get; set; }
        public string SupportExtensions { get; set; }
        public string CoverPath { get; set; }
        public string CoverFileExtensions { get; set; }
        public bool DeleteSourceFile { get; set; }
    }
    public class ArticleSettings
    {
        public const string SectionName = "ArticleSettings";
        public string FolderPath { get; set; }
        public string SupportExtensions { get; set; }
        public bool DeleteSourceFile { get; set; }
    }
    public class ActivitySettings
    {
        public const string SectionName = "ActivitySettings";

    }
    public class RepositorySettings
    {
        public const string SectionName = "RepositorySettings";
        public string FolderPath { get; set; }
        public string SupportExtensions { get; set; }
        public bool DeleteSourceFile { get; set; }
    }
}

File: ResourceManager/Settings/AppSettings.vb
Namespace ResourceManager.Settings
    Public Class AppSettings
        Public Const SectionName As String = "AppSettings"
        Public Property AppName As String
        Public Property AppVersion As String
        Public Property AppLanguage As String
        Public Property EbookSettings As EbookSettings
        Public Property ArticleSettings As ArticleSettings
        Public Property ActivitySettings As ActivitySettings
        Public Property RepositorySettings As RepositorySettings
    End Class
    Public Class EbookSettings
        Public Const SectionName As String = "EbookSettings"
        Public Property FolderPath As String
        Public Property SupportExtensions As String
        Public Property CoverPath As String
        Public Property CoverFileExtensions As String
        Public Property DeleteSourceFile As Boolean
    End Class
    Public Class ArticleSettings
        Public Const SectionName As String = "ArticleSettings"
        Public Property FolderPath As String
        Public Property SupportExtensions As String
        Public Property DeleteSourceFile As Boolean
    End Class
    Public Class ActivitySettings
        Public Const SectionName As String = "ActivitySettings"

    End Class
    Public Class RepositorySettings
        Public Const SectionName As String = "RepositorySettings"
        Public Property FolderPath As String
        Public Property SupportExtensions As String
        Public Property DeleteSourceFile As Boolean
    End Class
End Namespace
